### DISCUSSION TITLE
chain-maind.exe

### DISCUSSION THREAD

**Discussion by mukhwinder-s** [crypto-org-chain/chain-main#746] [General] [answered]
Hello,

i'm new in blockchain's development. 

I was reading the docs about configuring chain-maind, i've already downloaded the windows executable and done the first step.

At step 2-2 (https://crypto.org/docs/getting-started/mainnet.html#step-2-configure-chain-maind) is there a way to use the following function on windows to verify if genesis checksum it's ok?
![image](https://user-images.githubusercontent.com/96829843/161271525-42a2ca1e-6c5a-46f8-a54b-b3df6c9f88ea.png)

I got only two files in my working path , one is "config" and i created the and the second is the executable. Is this right?
![image](https://user-images.githubusercontent.com/96829843/161272269-ff749b80-349d-4a2f-9730-c2c80592e8eb.png)

p.s. the OS is Windows 11.
thanks for the support.

**Comment by tomtau** [comment, marked as answer]
On Windows, perhaps one option is PowerShell: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-filehash?view=powershell-7.2#example-1-compute-the-hash-value-for-a-file

**Reply by mukhwinder-s** [reply]
Thanks Tom,
i've just installed Linux CLI so i can work on that, on PowerShell is a lot different.

👍
